src/api/system.c

```diff
@@ -592,25 +592,25 @@ static int f_chdir(lua_State *L) {
   return 0;
 }
 
+static SDL_EnumerationResult list_dir_enumeration_callback(void *userdata, const char *dirname, const char *fname) {
+  (void) dirname;
```

Comment:
Will it return . and .. ?

Reply:
They're getting skipped:
https://github.com/libsdl-org/SDL/blob/93ac1e6894fac101008643a6cf0cc13b4524d18e/src/filesystem/posix/SDL_sysfsops.c#L66